scripts/main.js

```diff
@@ -192,19 +215,16 @@ function outputUI(note, parity, message, messageType, persistent = false) {
             infoString += ', ' + row + ' row) at beat ' + time.toFixed(3) + ':';
         }
         imgSrc += '.svg';
-
-        element.dataset.time = time.toFixed(3);
-        element.addEventListener('click', function () { scrollTo(time); });
```

Comment:
this is to allow scrolling to non-note events, it is a slightly inferior version of the outputUI in extra-warnings but it'll work for now